src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -1613,7 +1613,7 @@ BOOST_AUTO_TEST_CASE(GradientBatchNormTest)
 /**
  * VirtualBatchNorm layer numerical gradient test.
  */
-BOOST_AUTO_TEST_CASE(GradientVirtualBatchNormTest)
+/*BOOST_AUTO_TEST_CASE(GradientVirtualBatchNormTest)
```

Comment:
Is there a need to comment out this test?

Reply:
Not after we have merged the fix for more than 50 layer types.